pkgs/applications/networking/browsers/brave/update.sh

```diff
@@ -1,5 +1,19 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl gnused common-updater-scripts
+#!nix-shell -i bash -p curl gnused nix-update
 
-version="$(curl -sL https://brave-browser-apt-release.s3.brave.com/dists/stable/main/binary-amd64/Packages | sed -r -n 's/^Version: (.*)/\1/p' | head -n1)"
-update-source-version brave "$version"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+
+latestVersion="$(curl -sL https://brave-browser-apt-release.s3.brave.com/dists/stable/main/binary-amd64/Packages | sed -r -n 's/^Version: (.*)/\1/p' | head -n1)"
+latestVersionAarch64="$(curl -sL https://brave-browser-apt-release.s3.brave.com/dists/stable/main/binary-arm64/Packages | sed -r -n 's/^Version: (.*)/\1/p' | head -n1)"
+
+echo "Updating brave for x86_64-linux"
+nix-update --version "$latestVersion" \
```

Comment:
Maybe the script will require some tweaking.

My suggestion is to use a more elaborated script that does some template substitution.